create-employee.sql

```diff
@@ -0,0 +1,22 @@
+USE Kainoos_OrlaithM;
+
+SELECT * FROM Employee;
```

Comment:
This should probably be removed as you're trying to select data from a table you haven't created yet